integration-cli/docker_cli_plugins_test.go

```diff
@@ -148,7 +148,7 @@ func (s *DockerSuite) TestPluginInstallImage(c *check.C) {
 	testRequires(c, DaemonIsLinux, IsAmd64, Network)
 	out, _, err := dockerCmdWithError("plugin", "install", "redis")
 	c.Assert(err, checker.NotNil)
-	c.Assert(out, checker.Contains, "content is not a plugin")
+	c.Assert(out, checker.Contains, "target is image")
```

Comment:
Rather than change the test, we should probably fix the client to output a more descriptive message, like it does for docker pull:

https://github.com/docker/docker/blob/master/cli/command/image/pull.go#L77